app/assets/javascripts/discourse/app/templates/topic.hbs

```diff
@@ -105,7 +105,7 @@
         <PluginOutlet @name="topic-navigation" @connectorTagName="div" @args={{hash topic=this.model}} />
 
         {{#if info.renderTimeline}}
-          {{#if this.siteSettings.enable_glimmer_topic_timeline}}
+          {{#if this.siteSettings.enable_glimmer_topic_timeline this.experimentalTimelineLocalStorage}}
```

Comment:
Does this work? I would've thought we need

```suggestion
          {{#if (or this.siteSettings.enable_glimmer_topic_timeline this.experimentalTimelineLocalStorage)}}
```
But maybe there is some hbs syntax I wasn't aware of? 🤔